src/lib/vocab/membership/MembershipRepo.ts

```diff
@@ -67,6 +67,21 @@ export class MembershipRepo extends PostgresRepo {
 		return {ok: true, value: data};
 	}
 
+	async filterAccountMembersByCommunityId(
+		community_id: number,
+	): Promise<Result<{value: Membership[]}, ErrorResponse>> {
+		log.trace(`[filterByCommunityId] ${community_id}`);
+		const data = await this.db.sql<Membership[]>`
+		SELECT m.persona_id, m.community_id, m.created, m.updated 
+		FROM personas p JOIN (
+				SELECT persona_id, community_id, created, updated 
+				FROM memberships 
+				WHERE community_id=${community_id}
+			) as m ON m.persona_id = p.persona_id WHERE type = 'account';
```

Comment:
```suggestion
			) as m ON m.persona_id = p.persona_id WHERE p.type = 'account';
```

My SQL knowledge isn't great so this took longer to read without the `p.`, feel free to disregard.

Reply:
Good call!